Meeting translation (#13)
ropensci-org/champions-mentor-guidelines: mentorship-meetings.pt.qmd

# Reuniões de mentoria

> As reuniões de mentoria são as chamadas on-line individuais que você faz com o seu campeão.

Reviewer: ```suggestion
As reuniões de mentoria são as chamadas on-line individuais que você faz com a pessoa mentorada.
```

:::: callout-note

## Reunião presencial

> Esse formato de reunião é opcional. Não se espera que você tenha reuniões presenciais.

Reviewer: ```suggestion
Esse formato de reunião é opcional. Não esperamos que você tenha reuniões presenciais.
```

> :::

Author: ```suggestion

:::
```

## Agenda e anotações da reunião

> Pedimos que você faça anotações colaborativas das suas reuniões. Isso ajudará a concentrar a discussão e a manter um registro do progresso. Se você usa o Google Docs, pode criar um documento na pasta que criamos para você na unidade do programa. Você pode fazer uma cópia do documento [*modelo de reunião*](https://docs.google.com/document/d/1VtNYfDNYRS7xAbs1Ajf2S5wsv3mOQ0BcTn93cXAv6sc/) que fornecemos a você.

Reviewer: ```suggestion
Pedimos que faça anotações colaborativas das suas reuniões. Isso ajudará a concentrar a discussão e a manter um registro do progresso. Caso use o Google Docs, pode criar um documento na pasta que criamos para você no _drive_ do programa. Você pode fazer uma cópia do documento [*modelo de reunião*](https://docs.google.com/document/d/1VtNYfDNYRS7xAbs1Ajf2S5wsv3mOQ0BcTn93cXAv6sc/) que fornecemos.
```

## Frequência

> Você deve se reunir com a pessoa que está orientando. **todo mês** Estimamos reuniões de pelo menos 30 minutos, podendo haver mais sessões se necessário e se vocês dois puderem. O dia e a hora são combinados entre vocês. Consulte a seção @programação para ter uma ideia das ferramentas disponíveis para reuniões.

Reviewer: ```suggestion
Você deve se reunir **todo mês** com a pessoa que está orientando. Estimamos reuniões de pelo menos 30 minutos, podendo haver mais sessões se necessário e se ambas as pessoas puderem. O dia e a hora são combinados entre vocês. Consulte a seção @programação para ter uma ideia das ferramentas disponíveis para as reuniões.
```

Reviewer: @yabellini Does this section called @programação exist in Portuguese? Did you see that the link is not capturing the whole word? In Spanish it’s the same.

## Acompanhamento das reuniões

> Solicitaremos que você se registre em [este formulário](https://airtable.com/appF6OXmxkk8VmR8a/shrV3SruqtGwe1Bdj) quando você realizar essas reuniões. Isso nos ajuda a manter o controle e a saber se tudo está indo bem. **Você pode usar este formulário para solicitar apoio da equipe do rOpenSci.**.

Reviewer: ```suggestion
Solicitaremos que se registre utilizando [este formulário](https://airtable.com/appF6OXmxkk8VmR8a/shrV3SruqtGwe1Bdj) quando realizar essas reuniões. Isso nos ajuda a manter o controle e a saber se tudo está indo bem. **Você pode usar este formulário para solicitar apoio da equipe da rOpenSci.**.
```

> Na seção [*modelo de reunião do google doc*](https://docs.google.com/document/d/1VtNYfDNYRS7xAbs1Ajf2S5wsv3mOQ0BcTn93cXAv6sc/edit?usp=sharing) fornecemos um guia com os tópicos/questões que você pode usar em cada reunião. É claro que você pode escolher sua própria estrutura, mas é importante ter uma pauta.

Reviewer: ```suggestion
Na seção [*modelo de reunião do google doc*](https://docs.google.com/document/d/1VtNYfDNYRS7xAbs1Ajf2S5wsv3mOQ0BcTn93cXAv6sc/edit?usp=sharing) fornecemos um guia com os tópicos/questões que você pode usar em cada reunião. É claro que você pode escolher a sua própria estrutura, mas é importante ter uma pauta.
```

#### Primeira reunião

> Na primeira reunião, recomendamos que você

Reviewer: ```suggestion
Na primeira reunião, recomendamos:
```

> - Apresentações: para que você se conheça,

Reviewer: ```suggestion
- Apresentações para vocês se conhecerem melhor,
```

> - Um momento para os Campeões falarem sobre seus projetos com seus mentores.

Reviewer: ```suggestion
- Um momento para as Campeãs e os Campeões falarem sobre os seus projetos,
```

- Defina um ou dois objetivos para o seu projeto,

- Possíveis áreas para você pensar:
  

>   - *Para um novo projeto de pacote:* Uma primeira versão funcional do pacote seguindo as recomendações do guia de desenvolvimento do rOpenSci. Melhorar as funções, os testes, a documentação, o repositório e as funcionalidades. Gerar o site do pacote ou aprimorá-lo.

Reviewer: ```suggestion
  - *Para um novo projeto de pacote:* Uma primeira versão funcional do pacote seguindo as recomendações do guia de desenvolvimento da rOpenSci. Melhorar as funções, os testes, a documentação, o repositório e as funcionalidades. Gerar o site do pacote ou aprimorá-lo.
```

>   - *Para que um pacote seja enviado para revisão:* preparar o pacote para envio ao processo de revisão, enviar o pacote para revisão e concluir o processo. Melhorar a funcionalidade, os testes, a documentação, o repositório, a funcionalidade e a governança. Escreva um artigo para o JOSS ou JOSE sobre o pacote.

Reviewer: ```suggestion
  - *Para que um pacote seja enviado para a revisão:* preparar o pacote para envio ao processo de revisão, enviar o pacote para a revisão e concluir o processo. Melhorar a funcionalidade, os testes, a documentação, o repositório, a funcionalidade e a governança. Escreva um artigo para o JOSS ou JOSE sobre o pacote.
```

>   - *Para se tornar um revisor:* [preencha o formulário para revisar](https://airtable.com/app8dssb6a7PG6Vwj/shrnfDI2S9uuyxtDw) e depois revisar [a lista de pacotes que estão aguardando revisão](https://github.com/ropensci/software-review/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) e discuta quais podem ser boas opções para análise.  Informe ao Community Manager sobre essas preferências, que deverá entrar em contato com a equipe editorial para organizar a análise. [Leia o Guia de revisão de pacotes do rOpenSci](https://devguide.ropensci.org/es/index.es.html). Revisar pelo menos um pacote atribuído.

Reviewer: ```suggestion
  - *Para se tornar uma pessoa revisora:* [preencha o formulário para revisar](https://airtable.com/app8dssb6a7PG6Vwj/shrnfDI2S9uuyxtDw) e, em seguida, avalie [a lista de pacotes que estão aguardando revisão](https://github.com/ropensci/software-review/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) e discuta quais podem ser boas opções para análise. Informe sobre essas preferências a pessoa Gerente da Comunidade, que deverá entrar em contato com a equipe editorial para organizar a análise. [Leia o Guia de revisão de pacotes da rOpenSci](https://devguide.ropensci.org/es/index.es.html). Revisar pelo menos um pacote atribuído.
```

> - O que você espera do seu mentor?

Reviewer: ```suggestion
- O que esperar de uma pessoa mentora?
```

#### Entre as reuniões

> Para as reuniões seguintes, recomendamos que você

Reviewer: ```suggestion
Para as reuniões seguintes, recomendamos:
```

> - Destaques do mês anterior + Agradecimentos (alguém a quem você possa agradecer ou apreciar?)

Reviewer: ```suggestion
- Destacar os pontos do mês anterior e os agradecimentos (alguém a quem você possa agradecer ou apreciar?)
```

- Revisar os objetivos e o roteiro

> - Estamos progredindo e precisamos fazer algum ajuste?

Reviewer: ```suggestion
- Avaliar se estão progredindo e se precisam fazer algum ajuste
```

> - **Na reunião de janeiro/fevereiro** Definir um ou dois objetivos para as atividades de divulgação

Reviewer: ```suggestion
- **Na reunião de janeiro/fevereiro**, definir um ou dois objetivos para as atividades de divulgação
```

  
  - Analisar possíveis contribuições por tipo (eventos como "conversas da comunidade", webinars, reuniões, conferências (uma palestra ou workshop na useR!, Posit, LatinR), artigo de blog, tutorial, workshop, palestra, sessão de trabalho conjunto), por número, levar em conta quando e com quem (outra comunidade, grupo R-Ladies, universidade, instituições), por objetivo (desenvolver uma nova habilidade, tentar algo novo, praticar uma habilidade, organizar algo com outra comunidade).

> - **Quando é a convocação de um mês com uma coorte** Você pode fazer uma revisão da convocação da coorte: alguma reflexão, pergunta ou comentário após a última convocação?

Reviewer: ```suggestion
- **Quando ocorrer a convocação de um mês com a coorte**, você pode fazer uma revisão da convocação trazendo alguma reflexão, pergunta ou comentário.
```

- Em que mais podemos ajudar?

- Da próxima vez

> As reuniões intermediárias podem ser usadas para sessões de programação entre pares ou revisão de código.

Reviewer: ```suggestion
As reuniões intermediárias podem ser usadas para as sessões de programação entre pares ou para a revisão de código.
```
